packages/common/src/interfaces/ManifestJson.ts

```diff
@@ -47,8 +47,13 @@ export type ManifestJson = {
   thumbnail?: string
   /**
    * The dependencies of this project. Specify other projects that are to be installed alongside this one.
-   * @todo
-   * @example { "orgname/reponame": "0.1.2" }
+   * Can be either a string in the format 'namespace/project-name' or an object with detailed configuration.
+   * @example ["orgname/reponame", { "name": "orgname/another-repo", "commitHash": "abc123", "branch": "main" }]
```

Comment:
are we able to consider using `npm`'s git reference format of `git+${url}.git` eg. `git+https://github.com/ir-engine/ir-template.git`

https://docs.npmjs.com/cli/v9/commands/npm-install#description